scripts/testnet/sovereignBridge/config/esdt-safe.snippets.sh

```diff
@@ -1,5 +1,6 @@
 ESDT_SAFE_ADDRESS=$(mxpy data load --use-global --partition=${CHAIN_ID} --key=address-esdt-safe-contract)
 ESDT_SAFE_ADDRESS_SOVEREIGN=$(mxpy data load --use-global --partition=sovereign --key=address-esdt-safe-contract)
+NATIVE_ESDT=
```

Comment:
This looks weird.
Perhaps init it to some value anyway

Reply:
With .sh script I can just delete this. 
I added a check for nil in `updateSovereignConfig`